StripeConnect/StripeConnect/Source/Helpers/ConnectJSURLParams.swift

```diff
@@ -36,27 +36,27 @@ struct ConnectJSURLParams: Encodable {
 }
 
 extension ConnectJSURLParams {
-    init(component: ComponentType, apiClient: STPAPIClient) {
+    init(component: ComponentType, apiClient: STPAPIClient, publicKeyOverride: String?) {
         self.component = component
 
         // Validate that publishable key has been set
         STPAPIClient.validateKey(apiClient.publishableKey)
-
         if apiClient.publishableKeyIsUserKey {
-            // Dashboard app overrides
```

Comment:
nit: I think keeping this comment will help future devs understand the intention of this check